specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7197,6 +7292,51 @@
       },
       "description": "A skill to split a string into chunks of text."
     },
+    "CustomEntityLookupSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Text.CustomEntityLookupSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchIndexerSkill"
+        }
+      ],
+      "properties": {
+        "defaultLanguageCode": {
+          "$ref": "#/definitions/CustomEntityLookupSkillLanguage",
+          "description": "A value indicating which language code to use. Default is en."
+        },
+        "entitiesDefinitionUri": {
+          "type": "string",
+          "description": "Path to a JSON or CSV file containing all the target text to match against."
```

Comment:
Any auth requirements that should be documented? E.g. "Must be publicly accessible via a GET query."